components/inputs/input-number.js

```diff
@@ -58,7 +57,6 @@ class InputNumber extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(Lit
 		super();
 		this.autofocus = false;
 		this.disabled = false;
-		this.inputWidth = '4rem';
```

Comment:
We're going to want to apply this `4rem` width to the `<input type="text">` in order to keep the same size.

Reply:
Which... maybe means `<d2l-input-text>` needs to keep `input-width`?

Reply:
So if we keep this then adding an inline style of `width: 10rem` would set `input-number`'s container width to `10rem` and always default the `input-text`'s container `max-width` to `4rem`.

So that would mean that we would have to keep `input-width` for `input-number` as well because we would have to set both `style="width: 10rem;"` and `input-width="10rem"` inside the MVC control.